web/themes/custom/usagov/scripts/reportAProblem.js

```diff
@@ -276,64 +157,26 @@ var submitPressed = function () {
   "use strict";
 
   // reCaptcha Validation and error styling.
-  var reCaptchaValidationResult = reCaptchaValidation();
+  // var reCaptchaValidationResult = reCaptchaValidation();
```

Comment:
Please remove this line (and the comment above it), now that you know we don't need them. And also the commented-out line just before the return statement.